addons/mrp_landed_costs/tests/test_stock_landed_costs_mrp.py

```diff
@@ -179,3 +179,51 @@ def test_landed_cost_on_mrp_02(self):
         # Check that he can validate the landed cost without an access error
         landed_cost.with_user(stock_manager).button_validate()
         self.assertEqual(landed_cost.state, 'done')
+
+    def test_unbuild_MO_with_landed_cost(self):
+        consu_comp = self.env['product.product'].create({
+            'name': 'consu comp',
+            'type': 'consu',
+            'standard_price': 3.0,
+        })
+        final_prod = self.env['product.product'].create({
+            'name': 'final prod',
+            'type': 'product',
+            'categ_id': self.categ_all.id
+        })
```

Comment:
About data in a test
\- As a priority, use existing data from setup if possible
\- Create/process them in batch as much as possible